client/src/components/common/PageBanner.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import logo from '../../images/carrot_2.png';
+
+const PageBanner = (props) => {
```

Comment:
Our (Instacart) default for indentation for all languages is 2 spaces - adjust your editor and reformat.